Board gets frozen from time to time with:

[  152.160000] [DRV|VPU]/home/andrei/work/resin/resin-artik710/build/tmp/work-shared/artik710/kernel-source/drivers/media/platform/nxp-vpu/nx_vpu_api.c(536) : VPU Already Denitialized!!!

This can be avoided by not defining ENABLE_POWER_SAVING

Upstream-Status: Pending

Signed-off-by: Andrei Gherzan <andrei@resin.io>

Index: git/drivers/media/platform/nxp-vpu/nx_vpu_config.h
===================================================================
--- git.orig/drivers/media/platform/nxp-vpu/nx_vpu_config.h
+++ git/drivers/media/platform/nxp-vpu/nx_vpu_config.h
@@ -24,7 +24,6 @@
 
 /* VPU Clock Gating */
 #define	ENABLE_CLOCK_GATING
-#define	ENABLE_POWER_SAVING
 
 #define	NX_DBG_INFO			0
 
